internal/resources/fabric/service_token/resource_test.go

```diff
@@ -186,10 +227,10 @@ func testAccFabricZsideVirtualDeviceServiceTokenConfig(serviceTokenName string,
   			}
 
 		}
-    `, serviceTokenName, serviceTokenDescription, virtualDeviceUuid)
+    `, serviceTokenName, serviceTokenDescription, portUuid)
 }
 
-func testAccFabricAsidePortServiceTokenConfig(serviceTokenName string, serviceTokenDescription string, portUuid string) string {
+func testAccFabricZsidePortServiceTokenConfig(serviceTokenName string, serviceTokenDescription string, portUuid string) string {
```

Comment:
It's a little strange how the diff shows up in this file. Is this because you were attempting to organize the flow of the file?

Reply:
yes, I have re-organized the examples in alphabetical orders.